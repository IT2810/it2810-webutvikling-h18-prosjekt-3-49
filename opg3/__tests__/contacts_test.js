import React from 'react';
import ContactManager from "../Contacts";

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<ContactManager />).toJSON();
    expect(tree).toMatchSnapshot();
});