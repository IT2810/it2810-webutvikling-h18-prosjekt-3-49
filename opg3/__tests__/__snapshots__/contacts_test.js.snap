// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View>
    <View />
    <View>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        style={
          Object {
            "borderColor": "gray",
            "borderWidth": 1,
            "height": 40,
          }
        }
        value=""
      />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        style={
          Object {
            "borderColor": "gray",
            "borderWidth": 1,
            "height": 40,
          }
        }
        value=""
      />
      <View
        accessibilityComponentType="button"
        accessibilityLabel="accessibilityLabel er denne teksten"
        accessibilityTraits={
          Array [
            "button",
          ]
        }
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "padding": 8,
                  "textAlign": "center",
                },
                Object {
                  "color": "#4a4",
                },
              ]
            }
          >
            Add
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
